core/server/src/main/java/alluxio/master/file/FileSystemMasterClientRestServiceHandler.java

```diff
@@ -294,6 +301,67 @@ public Response mount(@QueryParam("path") String path, @QueryParam("ufsPath") St
     }
   }
 
+  private class MountPoint {
```

Comment:
if we ever want to add the getMountPoints endpoint to the Thrift interface, then we will also need a mechanism do deserialize it on the client side ... all other values sent over the wire are defined in the alluxio.wire package for that reason and I think it is important to be consistent

if the dependency on MountOptions is the only obstacle, then I suggest you inline its fields in the MountPoint object itself ... also instead of a constructor that takes MountInfo, you should provide a default constructor and setters and getter for all fields (similar to how other wire types do this)